scripts/eslint/tests/simple-indent.test.cjs

```diff
@@ -1,5 +1,5 @@
-import { RuleTester } from "./support/RuleTester";
-import rule = require("../rules/simple-indent");
+const { RuleTester } = require("./support/RuleTester.cjs");

```

Comment:
what determines whether a script becomes .cjs or .mjs?

Reply:
ESLint plugins have to be CJS (or at least ones provided by the "local" plugin), so these had to remain CJS; I'm just opting to encode that requirement in the filename rather than leaving them as `.js`.

Reply:
Renaming to be explicit makes sense, but I didn't see the pattern until you explained it. Thanks!

Reply:
Yeah, it's also helpful because I can gitignore all .js files in the script dir, which makes branch switching not completely messy when bisecting.